Documentation/k8s-object.md

```diff
@@ -0,0 +1,80 @@
+# Object Storage Quickstart
+
+Object storage exposes an S3 API to the storage cluster for applications to put and get data.
+
+### Prerequisites
+
+This guide assumes you have created a Rook cluster as explained in the main [Kubernetes guide](kubernetes.md)
+
+## Rook Client
```

Comment:
same here about duplicating the rook client instructions instead of linking them. it might be best to duplicate them here for readability (and because these steps are temporary), but i'm not sure.  your call.